New package: GeneralisedFilters v0.2.1

JuliaRegistrator opened the pull request:
- Registering package: GeneralisedFilters
- Repository: https://github.com/TuringLang/SSMProblems.jl
- Created by: @yebai
- Version: v0.2.1
- Commit: 24825ea991d4ad4b3a6c2ca6d6ddd2f8190dede9
- Reviewed by: @yebai
- Reference: https://github.com/TuringLang/SSMProblems.jl/commit/24825ea991d4ad4b3a6c2ca6d6ddd2f8190dede9#commitcomment-156007004
- Description: State space programming 
<!-- d8d2641e1edd8ccbea38203f7225e3680f063b5de557ff1f9a1f79186b7cdc501dc9e3fef9d5a0385803ac19c30c3dbc879a5aff44821e7fac5de7831283365ac92e3df52b1a20d2a5ceed6e5a63dc9c27bc91bc92d73f95aa5efdeaa37acc1734c8a3c3c41f3b0f1ecb506ff3e9199d1f6e4b897f9ab10b8c8396acd0200db7fcd50756686c2808dfa7875c50e867a8d9ee3aceef2afc8d03627ad7f322d44487442d8b18437428e1656cf4a5e0d0a9eeab2d33b7ff8f8e50007bd40e108a604e40092cff22d79e0c0c83ad268da867e5b02ce0a5d0f54b495446a6b148251874b40b7485723b8820a765d7d5f6603d -->

yebai commented:
cc @shravanngoswamii @THargreaves

goerz commented:
I just want to throw this out there, without wanting to step on the toes of any potential strong British identity you might have: I think the Julia community (or the wider computing community) if more or less centered around some version of "international English" that tends to match the American spelling of words, like Generalization, Optimization, Color, etc.

yebai commented:
There is a published paper using the name “GeneralisedFilters”, so we’d prefer to keep the name. 

[noblock]

goerz commented:
Alright, sounds good. If you at any point in the future want to alleviate the possible confusion of your international users w.r.t. the spelling, I made the (half-joking) recommendation in the Slack thread that you could create and register a second package `GeneralizedFilters` that simply re-exports everything from `GeneralisedFilters` (potentially with AE spelling aliases for function and type names). It would be relatively simple to maintain, but you'd have to synchronize releases.

[noblock]

JuliaRegistrator committed:
New package: GeneralisedFilters v0.2.1

UUID: 3ef92589-7ab8-43f9-b5b9-a3a0c86ecbb7
Repo: https://github.com/TuringLang/SSMProblems.jl.git
Tree: a4fc90fd9d3d2dfb8ff62196564482ecb8df3148

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
